.github/actions/bake-prod-stages/action.yml

```diff
@@ -0,0 +1,151 @@
+name: "Bake prod stages"
```

Comment:
Question: How much does baking prod/base vary? Is there a way they can share the majority of their code and use a input to flip the difference?

Reply:
The base stages don't have a workspace installed, while the prod stages do and are just stacked on top of the base stage layers. The fancy thing about the releaser stage is that it builds from the runner stage, so that you have a minimal prod image that only includes runtime dependencies, while the debugger includes the same workspace layer but build FROM the builder stage that includes build dependencies so you could rebuild/debug in production if also needed at the cost of greater image size.